### ISSUE TITLE
Maximum subscriptions allowed in courses

### ISSUE THREAD

**Issue by xam1234** [chamilo/chamilo-lms#2595]
Is there an option to set maximum number of students that may register in the course ?

**Comment by ywarnier** [comment]
@xam1234 just tested course registration password and it is working just fine. Make sure your course is in "open" visibility. Whenever students try to subscribe, it will ask for the password and not let them through if the password is incorrect.

**Comment by ywarnier** [comment]
@xam1234 regarding 1.11.8: we hope we'll be releasing a beta this month and the stable at the beginning of August. There are no plans to extend or integrate the advanced subscription plugin, there are a lot of other elements that have priority on this. Obviously, financial resources in the form of donations to the association or hiring an official provider to integrate it always help increase the speed of development by having more people available to work on it.

**Comment by xam1234** [comment]
@ywarnier  Did you test the password on version 1.11.6 or 1.11.x (11.chamilo.org) ?

**Comment by NicoDucou** [comment]
@xam1234 We always test on 1.11.x (11.chamilo.org) to know if the problem is present on an uptodate installation. So I would guess that this is what ywarnier did.

**Comment by ywarnier** [comment]
@xam1234 yes, tested with course 2497 with password "TEST" and user "mhalles/mhalles" (so don't use that one now to test because already subscribed): https://11.chamilo.org/main/auth/courses.php?category_code=ALL&pageCurrent=1&pageLength=12&search_term=&search_course=1 (don't mind the notice)
